src/util/time.h

```diff
@@ -36,9 +36,9 @@ class Time {
     // Returns the time elapsed since Mixxx started up in milliseconds.
     static qint64 elapsedMsecs() {
         if (s_testMode) {
-            return s_testElapsedTime / 1000;
+            return s_testElapsed_nsecs / 1000000;
         }
-        return s_timer.elapsed() / 1000;
+        return (s_timer.elapsed() / 1000000);
```

Comment:
Hm, interesting -- so this served to inflate time by 1000x which made soft takeover active essentially all of the time instead of only after 50 ms. I think turning this back on would be low risk then since it only serves to decrease how often ST ignores things.